freshmaker/config.py

```diff
@@ -414,7 +414,13 @@ class Config(object):
             'type': str,
             'default': '',
             'desc': 'The API URL of the Product Pages service'
-        }
+        },
+        'unpublished_exceptions': {
```

Comment:
Could you please add a method called `_setifok_unpublished_exceptions` that validates that the registry and repository keys are both set for any dictionary in this list?